rc/tools/go/gopls.kak

```diff
@@ -0,0 +1,98 @@
+# gopls.kak: gopls bindings for kakoune
+
+define-command -docstring "Enter gopls mode.
+Bindings for gopls in kakoune.
+
+Best used with mapping like:
+	map global normal '<whatever>' ': gopls<ret>'
+" \
+gopls %{ require-module gopls; evaluate-commands 'enter-user-mode gopls' }
+
+provide-module gopls %§
+
+evaluate-commands %sh{
+    if ! command -v gopls > /dev/null 2>&1; then
+        echo "fail Please install gopls or add to PATH!"
+    fi
+}
+
+try %{ declare-user-mode gopls }
```

Comment:
I'm not sure built-in script should declare user modes. The name might conflict with a user-declared user mode, plus having the editor pick mappings (even out of convenience) doesn't sound good. @mawww

Reply:
Ok, I will rework this part something like in git.kak, so there will be only a gopls command.